docs/source/cn/reduce_ops.py

```diff
@@ -0,0 +1,125 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.max,
+    r"""max(input, dim=None, keepdim=False) -> Tensor
+
+    返回一个包含 :attr:`input` 中元素最大值的 tensor.
```

Comment:
```suggestion
    返回 :attr:`input` 中的最大值。
```

标点符号也注意换成中文的，其它地方检查下。